src/gesv_mixed.cc

```diff
@@ -204,7 +210,9 @@ int64_t gesv_mixed(
             copy( R, X_lo, opts );
 
             // Solve the system A_lo * X_lo = R_lo.
+            Timer t_rgetrs;
```

Comment:
I have named this timer to be t_rgetrs due to comment above for solving the system: A_lo * X_lo = R_lo. However, would like input if this is an acceptable name.